### DISCUSSION TITLE
Write policy rule for group users

### DISCUSSION THREAD

**Discussion by ReactGuru007** [supabase/supabase#3647] [Questions] [answered]
Is it possible to write a policy for a group of users instead of just `uid() = created_by`?

for example, if I would like to have read access for people that are in the same team, right now I have to build a service for that. Would be nice to have the policy to take care of that. Is something like a function feasible for that. any insight is greatly appreciated.

users
id (primary)
account_id (foreign key)

accounts 
id (primary)

users from the same group can access their records.

**Comment by VictorPeralta** [comment, marked as answer]
You can see this example in the docs, I think that's what you're looking for. 

https://supabase.io/docs/guides/auth/row-level-security#policies-with-joins

**Reply by ReactGuru007** [reply]
thank you 👍
